custom-commands/silverstripe/sake

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
This should really have the normal annotations on it, https://ddev.readthedocs.io/en/latest/users/extend/custom-commands/#annotations-supported - at least add the name and such.

You may especially want `HostWorkingDir`.  It works for example with `ddev yarn`, executes in the relative directory with the root. You can look at the yarn command for an example.

Reply:
Yes, done, thank you!